src/components/Footer.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  const today = new Date();
+  const thisyear = today.getFullYear();
+</script>
+
+<footer>
+  <small>
+    &copy; {thisyear} kankodori. All rights reserved.
```

Comment:
nits

わざわざ変数使うまでもなさそう

```suggestion
</script>

<footer>
  <small>
    &copy; {new Date().getFullYear()} kankodori. All rights reserved.
```